apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -186,10 +186,14 @@ int ble_rmc_main(int argc, char *argv[])
 	RMC_LOG(RMC_TAG, "cmd : %s\n", argv[1]);
 
 	if (strncmp(argv[1], "init", 5) == 0) {
-		ret = ble_manager_init(&client_config, &server_config);
+		// ret = ble_manager_init(&server_config);
+		ret = ble_manager_init(NULL);
```

Comment:
how about to exit program if init is fail.

Reply:
Ok, I fixed it